vendor.conf

```diff
@@ -114,7 +114,7 @@ github.com/googleapis/gax-go v2.0.0
 google.golang.org/genproto 694d95ba50e67b2e363f3483057db5d4910c18f9
 
 # containerd
-github.com/containerd/containerd 63522d9eaa5a0443d225642c4b6f4f5fdedf932b
+github.com/containerd/containerd 08f7ee9828af1783dc98cc5cc1739e915697c667
```

Comment:
@tonistiigi shouldn't we sync `hack/dockerfile/install/containerd.installer` with this as well?

Reply:
We only use release versions in there. @dmcgowan will update that to `v1.1.1-rc.2` when it is released.

Reply:
Alright